src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/IntegrationTestBase.kt

```diff
@@ -157,9 +159,31 @@ abstract class IntegrationTestBase {
       .isCreated()
   }
 
-  fun getInduction(prisonNumber: String): CiagInductionResponse =
-    webTestClient.get()
+  fun createInduction(
```

Comment:
I've added this for an upcoming `GetInductionTest`, which I'll add in my next PR (I meant to exclude it from this PR, but never mind!).